dev/com.ibm.ws.microprofile.config_fat/.classpath

```diff
@@ -19,11 +19,10 @@
 	<classpathentry kind="src" path="test-applications/types.war/src"/>
 	<classpathentry kind="src" path="test-applications/converterApp/src"/>
 	<classpathentry kind="src" path="test-applications/converterApp/resources"/>
-	<classpathentry kind="src" path="test-applications/cdiConfig.war/src"/>
-	<classpathentry kind="src" path="test-applications/defaultSources.war/src"/>
-	<classpathentry kind="src" path="test-applications/earlib.jar/src"/>
-	<classpathentry kind="src" path="test-applications/warVisibility.war/src"/>
-	<classpathentry kind="src" path="test-applications/sharedLibUser.war/src"/>
+	<classpathentry kind="src" path="test-applications/defaultSources.war/resources"/>
```

Comment:
Since there is no java source in this directory, I didn't think we included it in .classpath.  At least other projects that have 'resources' directories with just XML have not done this.

Reply:
Since there are other applications in this one I am leaving these ones in.